infrastructures/infrastructure-azure/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AzureInfrastructure.java

```diff
@@ -52,7 +53,7 @@
 
     private static final String DEFAULT_RM_HOSTNAME = "localhost";
 
-    private final static int PARAMETERS_NUMBER = 23;
+    private final static int PARAMETERS_NUMBER = 24;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)